users/csc027/csc027.c

```diff
@@ -0,0 +1,63 @@
+#include "csc027.h"
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    static const char* git_macros[] = {
```

Comment:
You could probably move this out of the function.

Reply:
Makes sense.